packages/core/src/config/parse.ts

```diff
@@ -857,15 +884,14 @@ export const parseAndValidateVariable = (
 const parseContractVariables = (
   contractConfig: UserContractConfig,
   storageLayout: SolidityStorageLayout,
-  compilerOutput: CompilerOutput
+  compilerOutput: CompilerOutput,
+  cre: ChugSplashRuntimeEnvironment
 ): {
-  [name: string]: ParsedConfigVariable
+  variables: ParsedConfigVariables
```

Comment:
> I think this can be simplified to `return parsedConfigVariables`

Similarly, I think this can be simplified from `{ variables: ParsedConfigVariables }` to just `ParsedConfigVariables`